src/porepy/models/geometry.py

```diff
@@ -711,8 +711,11 @@ def outwards_internal_boundary_normals(
         # signs, and project back up to all subdomains.
         flipped_normals = flip @ primary_face_normals
         # Project to mortar grid, as a mapping from mortar to the subdomains and back
-        # again.
-        outwards_normals = mortar_projection.primary_to_mortar_avg @ flipped_normals
+        # again. If we are to use cell volumes from interfaces to normalize, projection
+        # must logically be integration, not average. This also means that the normals
+        # have length equal to cell size on mortar grids, by analogy to face area for
+        # subdomains.
+        outwards_normals = mortar_projection.primary_to_mortar_int @ flipped_normals
```

Comment:
Do you agree with the comment and the change, @keileg?

Reply:
I think it makes sense.

Reply:
To me this text is not clear, for example, I am unsure if you refer to `cell volumes` as the cell geometrical measurement (length, area, volume).  What do you mean by the length of a normal? do you mean its magnitude or an array dimension?